tests/unit/UI/Buttons/IconButton.spec.js

```diff
@@ -0,0 +1,16 @@
+import { mount } from "@vue/test-utils";
+import IconButton from "@/components/UI/Buttons/IconButton";
+
+describe("IconButton.vue", () => {
+  it("renders title correctly", () => {
+    const buttonText = "Test Button";
+    const wrapper = mount(IconButton, {
+      props: {
+        titleConfig: {
+          value: buttonText,
+        },
+      },
+    });
+    expect(wrapper.text()).toMatch(buttonText);
```

Comment:
where can we see the outputs of some of these things while developing - say, by using console.log here?

Reply:
Yep. `console.log()` works. Here's an output of this line of code `console.log(wrapper.find('option').html());`

![image](https://user-images.githubusercontent.com/12053186/122271033-97469d80-cefc-11eb-92ed-49a77727a811.png)